content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst

```diff
@@ -59,15 +61,23 @@ clicking :guilabel:`Tickets` on the team's card (or by going to the Projects das
 Link the Helpdesk ticket to the SO
 ==================================
 
-To link the :abbr:`SO (Sales Order)` to the ticket, start on the ticket form, click
-:guilabel:`Edit`, and then, click the :guilabel:`Sales Order Item` field. Next, select the
-:abbr:`SO (Sales Order)` that was created earlier from the drop-down menu. Lastly, click
-:guilabel:`Save` to connect the ticket and the :abbr:`SO (Sales Order)`.
+Once a timesheet is recorded, the :guilabel:`Sales Order Item` field will appear on the ticket form.
+To link the :abbr:`SO (Sales Order)` to the ticket, start on the ticket form and click
+:guilabel:`Edit`. Next, select the :abbr:`SO (Sales Order)` that was created earlier from the
+drop-down menu. Odoo will automatically filter the sales orders to only show sales orders that are
+connected to the ticket's customer. Lastly, click :guilabel:`Save` to connect the ticket and the
+:abbr:`SO (Sales Order)`.
 
-.. image:: reinvoice_from_project/link-so-ticket.png
+.. image:: reinvoice_from_project/link-soi-to-ticket.png
    :align: center
    :alt: Link the SO item to the ticket.
 
+.. tip::
+   If a timesheet entry is recorded, but the agent does not want to bill the client for that time,
+   go to the :guilabel:`Timesheets` tab and toggle on the visibility of the :guilabel:`Sales Order
+   Item` column. Then, make sure all billable timesheet entries have the correct sales order item
+   selected and leave the non-billable timesheet entry's :abbr:`SOI (Sales Order Item)` blank.
```

Comment:
Got it, I will take out the part about entering the correct SOI for each billable entry. I also will add that this can be used for billing at different rates.